doc/pip_and_pep_668.rst

```diff
@@ -0,0 +1,72 @@
+Pip installation after PEP 668
+==============================
+
+`PEP-668`_ introduced `externally managed environments <externally-managed-environments>`_ to Python packaging.
+
+``rosdep`` is designed to use pip as an alternative system package manager, rosdep installation of pip packages requires installing packages globally as root.
```

Comment:
this document is a bit inconsistent with rosdep and `rosdep`, but I'll let you decide if you want to change anything.

Reply:
My mental style guide is `rosdep` when referring to the CLI tool and rosdep when referring to the project. I am always tempted into using `rosdep` in appropriately when its the first word of the sentence because Rosdep and ROSdep are both wrong.

I will reword to avoid the entire problem :D (that mainly exists in my head)

Reply:
Actually, I'm just going to get over myself and start the sentence with the proper lowercase noun. It's the 2020s.